src/classes/frWSCampaignController.cls

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *  Copyright (c) 2016, Funraise Inc
+ *  Copyright (c) 2018, Funraise Inc
```

Comment:
Might as well update it to 2019 since we are in here

Reply:
Is that how it's supposed to work? I just set them to the year I know they were created (I think the original donor/donation classes still say something like 2016).  I think we should have a separate branch to update all of that legal stuff

Reply:
We should probably create a LICENSE file too in the project root directory, I could see a separate branch handling that along with these comments

https://help.github.com/en/articles/licensing-a-repository